sdk/communication/communication-job-router/src/routerClient.ts

```diff
@@ -382,62 +367,45 @@ export class RouterClient {
    * @param options - Operation options.
    */
   public async createJob(
-    request: RestCreateJobRequest,
+    request: RouterJob,
     options: CreateJobOptions = {}
-  ): Promise<CreateJobResponse> {
-    return this.client.jobRouter.createJob(request, options);
+  ): Promise<RouterJob> {
+    return this.client.jobRouter.createJobV2(request, options);
   }
 
   /**
-   * Gets the list of enqueued jobs.
-   * @param queueId - The queue id.
-   * @param options - List jobs options.
+   * Gets an job.
+   * Returns job with the id of the job.
+   * @param jobId - The id of the job to get.
+   * @param options - Operation options.
    */
-  public listEnqueuedJobs(
-    queueId: string,
-    options: ListEnqueuedJobsOptions = {}
-  ): PagedAsyncIterableIterator<RouterJob> {
-    return this.client.jobRouter.listEnqueuedJobs(queueId, options);
+  public async getJob(
+    jobId: string,
+    options: GetJobOptions = {}
+  ): Promise<RouterJob> {
+    return this.client.jobRouter.getJob(jobId, options);
   }
 
   /**
    * Update or insert labels of a job by Id.
-   * @param jobId - The ID of the job to update.
-   * @param labels - The labels to update.
+   * @param updatedModel - The updating job
+   * @param forceClassification - If forcing classification after update
    * @param options -  Operation options.
    */
   public async updateJobLabels(
-    jobId: string,
-    labels: { [propertyName: string]: LabelValue },
+    updatedModel: RouterJob,
+    forceClassification?: boolean,
     options: UpdateJobLabelsOptions = {}
   ): Promise<RouterJob> {
-    return this.client.jobRouter.updateJobLabels(jobId, labels, options);
-  }
+    options.patch = updatedModel;
+    if (!updatedModel.id) {
+      throw new Error("Updated model must contain id")
+    }
+    if (forceClassification) {
+      options.forceClassification = forceClassification;
+    }
 
-  /**
-   * Updates an existing job by Id and forcing it to be reclassified.
-   * @param jobId - The ID of the job to update.
-   * @param request - The update and classify job request.
-   * @param options -  Operation options.
-   */
-  public async reclassifyJob(
-    jobId: string,
-    request: RestReclassifyJobRequest,
-    options: UpdateJobClassificationOptions = {}
-  ): Promise<RouterJob> {
-    return this.client.jobRouter.reclassifyJob(jobId, request, options);
-  }
-
-  /**
-   * Updates an existing job's queueId, priority, requiredAbilities and labels.
-   * @param jobId - The ID of the job to update.
-   * @param options - UpdateClassification options encapsulating the queueId, priority and worker requirements.
-   */
-  public async updateJobClassification(
```

Comment:
We should not remove this SDK route.  Please rename it to QueueJob and refactor it to use UpdateJob internally.

Reply:
Will refactor it